test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -528,14 +532,110 @@ def expected_seeded_manifest(self):
                     'description': '',
                     'columns': {},
                 },
+                'test.test.not_null_model_id': {
+                    'alias': 'not_null_model_id',
+                    'column_name': 'id',
```

Comment:
cool! We can reverse-engineer the model that's being tested by checking out the first value 
 in `node['depends_on']['nodes']`. That's not ideal long-term, but I think it's perfectly fine for our first cut of this

Reply:
should we just include the target model in the manifest? At this point we've really made test manifests and model manifests into different things, maybe we should have two different schemas.

Reply:
I think having two different schemas long-term is a really good idea, but I defer to @cmcarthur on if this is a good time to do it.

Schema tests are generally one-to-one with models, so I buy adding a `model` field to the test nodes. One thing that we'd need to account for are "custom data tests" -- these can really test 0, 1, or many models. I'm cool with adding the target model if it's not too much work for just schema tests.

Reply:
Yeah, it's pretty easy. We can do it in a new PR, too. As we've worked on this the various "Node schemas" have begun to diverge pretty heavily (seeds having the silly `-- csv --` in their raw_sql comes to mind too). I think doing a full separation is a lot, but just a separate schema for tests should be pretty simple - JSONSchema has this `anyOf` concept that should make it pretty straightforward.

Reply:
I am totally on board with the idea of both diverging the schemas for different node types, and including the "target" unique id in the test schema